### DISCUSSION TITLE
RAI Italia

### DISCUSSION THREAD

**Discussion by ylnest2018** [iptv-org/community#1122] [Q&A]
I have checked the following playlists as per the channel list csv, looking for some help

RaiItaliaAfrica.it - https://iptv-org.github.io/iptv/regions/afr.m3u - Channel does not exist
RaiItaliaAmerica.it - https://iptv-org.github.io/iptv/regions/amer.m3u - Channel exists but broken
RaiItaliaAsia.it - https://iptv-org.github.io/iptv/regions/sea.m3u - Channel exists but broken
RaiItaliaAustralia.it - https://iptv-org.github.io/iptv/regions/oce.m3u - Channel does not exist

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
